sdk/communication/Azure.Communication.Identity/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
 ## 1.1.0-beta.1 (Unreleased)
-
+- Added ExchangeAccessToken operation
```

Comment:
I think we should expand it a bit - Added ExchangeAccessToken to provide the ability to exchange M365 tokens for a communication token